account_move_line_accounting_description/models/account_move.py

```diff
@@ -7,11 +7,10 @@
 class AccountMoveLine(models.Model):
     _inherit = "account.move.line"
 
-    external_name = fields.Char(string="External Name")
+    external_name = fields.Char()
 
     @api.onchange("product_id")
     def _onchange_product_id(self):
-        super()._onchange_product_id()
```

Comment:
If on v16 the method `_onchange_product_id()` does not exists, it is OK to remove the inheritance, but please use a more specific name for it, not such a generic one.